addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14637,12 +14646,12 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36210"
-msgid "Prevent the 'connection lost' notification window from displaying when the PVR backend server is not available."
+msgid "Prevent the \"connection lost\" notification window from displaying when the PVR backend server is not available."
```

Comment:
Actually it refers to the setting msgctxt "#36030" so it is a capitalized entry and the reason why is quoted.